components/actionbutton/stories/actionbutton.test.js

```diff
@@ -76,5 +74,9 @@ export const ActionButtonGroup = Variants({
 		testHeading: "Disabled + selected",
 		isDisabled: true,
 		isSelected: true,
+	}, {
+		testHeading: "Quiet + emphasized",
+		isEmphasized: true,
+		isQuiet: true,
```

Comment:
I would probably move this to the `testData`, instead of `stateData`. Stepper does something similar with its `Quiet + invalid` test case.

<img width="408" alt="Screenshot 2024-09-23 at 10 28 51 AM" src="https://github.com/user-attachments/assets/2134e0c8-01de-4f33-b725-75172c2e44fc">

If we move this to `testData`, then we could see the visual styles for selected, hovered, etc too. 
<img width="1313" alt="Screenshot 2024-09-23 at 10 31 06 AM" src="https://github.com/user-attachments/assets/28f6d91e-d0d5-40c8-90a2-afaf64fdcc5a">